routes/tasks.js

```diff
@@ -145,6 +145,8 @@ router.post('/:user_id', function(req, res, next) {
     mangoTransactions: [],
     subTasks: [],
     isDone: false,
+    completionTimestamp: null,
+    startTimestamp: Date.now(),
```

Comment:
is a bit redundant since the already existing timestamp can serve whatever function startTimestamp is needed for